src/Template.php

```diff
@@ -689,112 +614,87 @@ public function loadTemplateFromString($str)
 
     /**
      * Used for adding unique tag alternatives. E.g. if your template has
-     * {$name}{$name}, then first would become 'name#1' and second 'name#2', but
+     * {$name}{$name}, then first would become 'name#0' and second 'name#1', but
      * both would still respond to 'name' tag.
      *
      * @var array
      */
-    private $tag_cnt = [];
+    private $tagCnt = [];
 
     /**
-     * Register tags and return unique tag name.
-     *
-     * @param string $tag tag name
-     *
-     * @return string unique tag name
+     * Register tag and return unique tag name.
      */
-    protected function regTag($tag)
+    protected function regTag(string $tag): string
     {
-        if (!isset($this->tag_cnt[$tag])) {
-            $this->tag_cnt[$tag] = 0;
+        if (!isset($this->tagCnt[$tag])) {
+            $this->tagCnt[$tag] = -1;
```

Comment:
can we keep this as 0 for BC?

Reply:
no longer avaiable to outside, thus irrelevant